internal/provider/data_source_docker_plugin.go

```diff
@@ -17,12 +17,15 @@ func dataSourceDockerPlugin() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
Ideally, this should be the other PR.

Reply:
I know, but after updating the sdk, the tests failed because the attribute was missing. I guess the old sdk just covered this. This is why I added the property

Reply:
I see. Thank you for your explanation.